doc/Channels.md

```diff
@@ -56,36 +56,36 @@ Each channel is defined by these values:
 Additional keywords for the full text search
 ---
 
-Additionally to the search for content, there are additional keywords that can be used in the full text search:
+Additionally to the search for content, there are additional keywords that can be used in the full text search. Alternatives are presented via "|".
```

Comment:
```suggestion
Additionally to the search for content, there are keywords that can be used in the full text search.
Alternatives are presented with "|".
```

Reply:
Changes are done and also the merge commit is solved.